src/meili/candidate_search.cc

```diff
@@ -44,17 +44,17 @@ CandidateCollector::WithinSquaredDistance(const midgard::PointLL& location,
       continue;
     }
 
-    // Get the edge and its opposing edge. Transition edges are not
-    // allowed so we do not need to check node levels.
-    const auto opp_edgeid = reader_.GetOpposingEdgeId(edgeid, tile);
```

Comment:
this was a bug. when you get the opposing id with the tile of the regular edge it can flip modify the tile to be the wrong tile which we use again to refer to the original edge. the code was throwing an exception, getting caught and failing the map match. this would happen on all map match input points which have radii large enough to intersect an edge that crosses a tile boundary

Reply:
cc @PureW @mandeepsandhu @mookerji